resources/views/cities.blade.php

```diff
@@ -72,5 +74,79 @@ class="del-btn ml-2 dark:bg-red-600 hover:bg-red-400">
     <div class="mt-12 mb-4 px-12">
         {{ $cities->links() }}
     </div>
+
+    <script>
+        $(document).ready(function () {
+
+          function displayModal(title, content) {
+              var modal = document.getElementById('modal')
+
+              $('[modal-title]').text(title)
+              $('[modal-content]').html(content)
+
+              modal.showModal()
+          }
+
+          function closeModal() {
+              var modal = document.getElementById('modal')
+              modal.close()
+          }
+
+          $('[close-modal-btn]').click(closeModal)
+
+          $('button[create-button]').click(function() {
+              var city = {
```

Comment:
Always declare variables with let. https://developer.mozilla.org/es/docs/Web/JavaScript/Reference/Statements/let

But in this case you can declare it with `const city`